### DISCUSSION TITLE
Collection Log "!" Command

### DISCUSSION THREAD

**Discussion by StillNoTbow** [runelite/runelite#15178] [Ideas]
Would it be possible to get a runelite command to show current collection log # similar to the !Pets and !kc commands?

Either show
Collection Log: 255/1404 
or
Collection Log: 255

**Comment by BudLightning** [comment]
Additionally !clog barrows -> showing x/25 etc.. I think this would be really cool.

**Reply by FraserGreen** [reply]
This along with images like in !pets would be sick

**Comment by Zaino60** [comment]
Support! Would be amazing if this could be added since it doesn't look like Jagex is adding Collection Log Hi-scores anytime soon

**Comment by nallepuhi** [comment]
We have this already

**Comment by EwyBoy** [comment]
Maybe with a completion percentage like:
`Collection Log: 255/1404 - 18.16% completed`

**Comment by IromTypo** [comment]
This is in the plugin hub, it's called Collection Log. Use the command !log or !log [log]. You can also upload your log to collectionlog.net :)
